### ISSUE TITLE
Recreate container

### ISSUE THREAD

**Issue by dbrgn** [softprops/shiplift#79]
I'm trying to write a tool that pulls an image and recreates an existing container.

To create a container, I need to provide a `ContainerOptions` instance.

Is there a way to extract the `ContainerOptions` from an existing container by inspecting it? Could we maybe `impl From<ContainerDetails> for ContainerOptions`?

**Comment by softprops** [comment]
That approach sounds very cool

**Comment by TECHNOFAB11** [comment]
I'm trying to do the same and am new to Rust. Did you find a way to do it? In Python i just used the data from inspect, removed some keys and moved everything in the "Config" key up to the main dict. But i now tried for hours and cant seem to get shiplift to just accept a HashMap or create ContainerOptions out of it

**Comment by dbrgn** [comment]
> Did you find a way to do it?

No, I started using [watchtower](https://hub.docker.com/r/containrrr/watchtower) instead 😄
